### PR TITLE
fix title, format code

### PR DESCRIPTION
**Pull request by RushilJalal**
Hey @0xabdulkhalid 
These are the changes I made:
1. Fix the title from plibary to plibrary
2. Format code using Prettier
Kindly review and merge🙏
Thanks

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "plibrary",
+  "lockfileVersion": 3,
+  "requires": true,
+  "packages": {}
+}
```
Comment (0xabdulkhaliq) [inline package-lock.json]: You don't need to commit `package-lock.json` file, you can remove this one.

**Review by 0xabdulkhaliq — CHANGES_REQUESTED** [review]

File: package-lock.json (hunk 1 above)
Comment (RushilJalal) [reply package-lock.json]: I deleted the file. Do I need to create another commit now?
Sorry I'm new...

**Review by RushilJalal — COMMENTED** [review]
Deleted `package-lock.json`

File: package-lock.json (hunk 1 above)
Comment (0xabdulkhaliq) [reply package-lock.json]: It's okay don't worry about, We can only learn things by doing it.

You have done a great job!

**Review by 0xabdulkhaliq — APPROVED** [review]
LGTM 👀

**Comment by 0xabdulkhaliq** [comment]
Thank you for your contribution @RushilJalal !

**Pull request merged**